weblate/Dockerfile

```diff
@@ -23,6 +23,9 @@ RUN env DEBIAN_FRONTEND=noninteractive apt-get update \
   && apt-get clean
 
 # Examples
+RUN curl -L https://github.com/github/hub/releases/download/v2.2.9/hub-linux-amd64-2.2.9.tgz | tar xzv && \
+  cp hub-linux-amd64-2.2.9/bin/hub /usr/bin
+
```

Comment:
This way you leave all the files in the docker image, what is IMHO not a good idea. Doing something like `tar xzv --wildcards hub-linux*/bin/hub` should do better job. Also you should remove the hub-linux-amd64-2.2.9 directory after copying the file out.